terraform/aks/application.tf

```diff
@@ -11,7 +11,7 @@ module "application_configuration" {
 
   secret_variables = {
     DATABASE_URL        = local.database_url
-    BLAZER_DATABASE_URL = module.postgres.url
+    BLAZER_DATABASE_URL = module.database.url
```

Comment:
This should be the same as DATABASE_URL, ie local.database_url

Reply:
Yeah I spotted that, thanks